docker-compose/run_integration_tests_local_keybase.sh

```diff
@@ -0,0 +1,199 @@
+  exit 1
+fi
+if [ -z "${DJANGO_USER_ID_ITERATIONS}" ]
+then
+  echo_msg "ERROR: The DJANGO_USER_ID_ITERATIONS variable was not sourced!"
+  exit 1
+fi
+
+# Check temp certstore dir and create if not existing
+if [ -d "${CERTSTORE_TEMPORARY_MOUNT_PATH}" ]
+then
+  echo_msg
+  echo_msg "  - OK: The temporary certstore mount path is found at: ${CERTSTORE_TEMPORARY_MOUNT_PATH}"
+else
+  mkdir ${CERTSTORE_TEMPORARY_MOUNT_PATH}
+  echo_msg
+  echo_msg "  - OK: Created the temporary certstore mount path at: ${CERTSTORE_TEMPORARY_MOUNT_PATH}"
+fi
+
+# Keybase cert files
+keybase_certfiles="${keybase_env_path}/${KEYBASE_CERTFILES_SUBPATH}/${KEYBASE_CERFILES_SUBPATH}"
+keybase_cert_file="${keybase_certfiles}/${CERT_FILENAME}"
+keybase_key_file="${keybase_certfiles}/${KEY_FILENAME}"
+
+# Check that certfiles in keybase dir exist
+if [ -f "${keybase_cert_file}" ]
+then
+  echo_msg
+  echo_msg "  - OK: The cert file was found at: ${keybase_cert_file}"
+else
+  echo_msg
+  echo_msg "ERROR: The cert file could NOT be found at: ${keybase_cert_file}"
+  exit 1
+fi
+if [ -f ${keybase_key_file} ]
+then
+  echo_msg
+  echo_msg "  - OK: The key file was found at: ${keybase_key_file}"
+else
+  echo_msg
+  echo_msg "ERROR: The key file could NOT be found at: ${keybase_key_file}"
+  exit 1
+fi
+
+# Copy certfiles from KeyBase to local for container mount
+echo_msg "  - COPY certfiles from KeyBase to local temp for container mount..."
+echo_msg
+cp ${keybase_cert_file} "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.cert.pem"
+cp ${keybase_key_file} "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.key.nocrypt.pem"
+
+# Check if tests are to run in docker-compose
+if [[ $1 == "dc" ]]
+then
+  # Stop web if running and restart db
+  docker-compose stop web
+  docker-compose restart db
+
+  # Run docker-compose containter one-off
+  echo_msg
+  echo_msg "------RUNNING DOCKER-COMPOSE CONTAINER WITH INTEGRATION TESTS------"
+  echo_msg
+  echo_msg "    INTEGRATION_TESTS_LIST: ${INTEGRATION_TESTS_LIST}"
+  echo_msg
+  docker-compose run \
+    --service-ports \
+    -e HOSTNAME_URL=${HOSTNAME_URL} \
+    -e DJANGO_USER_ID_SALT=${DJANGO_USER_ID_SALT} \
+    -e DJANGO_USER_ID_ITERATIONS=${DJANGO_USER_ID_ITERATIONS} \
+    -e DJANGO_FHIR_CERTSTORE=${DJANGO_FHIR_CERTSTORE} \
+    -v "${CERTSTORE_TEMPORARY_MOUNT_PATH}:${DJANGO_FHIR_CERTSTORE}" \
+    web bash -c "python runtests.py --integration ${INTEGRATION_TESTS_LIST}"
+fi
+
+# Check if tests are to run in docker CBC container one-off
+if [[ $1 == "cbc" ]]
+then
+  echo_msg
+  echo_msg "------RUNNING DOCKER CONTAINER CBC IMAGE WITH INTEGRATION TESTS------"
+  echo_msg
+  echo_msg "    INTEGRATION_TESTS_LIST: ${INTEGRATION_TESTS_LIST}"
+  echo_msg
+  docker run \
+    -e CERT_FILE=${CERT_FILE} \
+    -e KEY_FILE=${KEY_FILE} \
+    -e DJANGO_USER_ID_SALT=${DJANGO_USER_ID_SALT} \
+    -e DJANGO_USER_ID_ITERATIONS=${DJANGO_USER_ID_ITERATIONS} \
+    -e DJANGO_FHIR_CERTSTORE=${DJANGO_FHIR_CERTSTORE} \
```

Comment:
```suggestion
    -e DJANGO_FHIR_CERTSTORE=${DJANGO_FHIR_CERTSTORE} \
    -e FHIR_URL=${FHIR_URL} \
```

I had to pass through the FHIR_URL var to get this working locally, otherwise it uses the default in base.py.

Reply:
@njdister A recent change I made broke this. Just fixed it :-)